export.py

```diff
@@ -12,18 +12,20 @@
 from collections import namedtuple
 from functools import reduce
 
-from selenium import webdriver
+from seleniumwire import webdriver
```

Comment:
[seliumwire](https://github.com/wkeeling/selenium-wire) isn't maintained. Can we stick with selenium?